packages/cozy-sharing/locales/en.json

```
@@ -211,13 +228,21 @@
       "sharedByMe": "Shared",
       "sharedWithMe": "Shared with me",
       "shareByLink": {
-        "subtitle": "By public link",
+        "subtitle": "Or share by link",
         "desc": "Anyone with the provided link can see and download your photos.",
         "fetchFailed": "Woops! It seems your connectivity is limited, try again later when it gets better.",
-        "creating": "Creating your link...",
+        "create": "Generate a link",
+        "activated": "Link activated",
         "copy": "Copy link",
         "copied": "Link has been copied to clipboard",
-        "failed": "Unable to copy to clipboard"
+        "failed": "Unable to copy to clipboard",
+        "ro": "Can read",
+        "rw": "Can change",
+        "desc": {
+          "ro": "Link holders can only read or download content.",
+          "rw": "Link holders can alter and delete content."
```

```suggestion
          "rw": "Link holders can change and delete content."
```

Are we sure about `alter`? We use `change` or `edit` in other places.